src/action_binding.rs

```diff
@@ -18,10 +18,24 @@ use crate::{
     trigger_tracker::TriggerTracker,
 };
 
-/// Bindings of [`InputAction`] for [`Actions`](crate::actions::Actions).
+/// Bindings associated with an [`InputAction`] marker.
 ///
-/// These bindings are stored separately from [`ActionMap`] to allow a currently
-/// evaluating action to access the state of other actions.
+/// Stored inside [`Actions`](crate::actions::Actions).
+///
+/// Bindings are stored separately from [`ActionMap`] to allow reading other actions' data during evaluation.
+/// The current action's data updates at evaluation's end.
```

Comment:
When does evaluation end? Where do I read more about this?

Reply:
Let's remove this completely, I will include this information into the evaluation flow below.